Config-Files/C&C++/Windows/tasks.json

```diff
@@ -51,4 +51,4 @@
       }
     }
   ]
-}
\ No newline at end of file
+}

```

Comment:
and its better to avoid this type of changes like adding a new line in file :3 @Mahdi-Hasan